### PR TITLE
All the firstFloor jazz

### LINKED ISSUE #38: Create first floor code
Using `levels/ground.go` as a template, create a `levels/first.go` to load the first floor.

Test this by changing line `gamestate.SetLevel(levels.Ground)` in `main.go` (currently line 46), so it loads `levels.First` instead - but don't commit that change

### PR DESCRIPTION
**Pull request by PhantomRaviolis**
Resolves #38

### CODE REVIEW

File: levels/first.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+const (
+	firstImagePath          = "assets/graphics/firstFloor.png"
+	firstCollisionPath      = "assets/csv/firstFloorWalls.csv"
+	firstActivationZonesCSV = "assets/csv/firstFloorZones.csv"
+)
+
+var (
+	// First is the ground level
+	First = &Level{
+
```
Comment (bcvery1) [inline levels/first.go]: Remove blank line

File: levels/first.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+)
+
+const (
+	firstImagePath          = "assets/graphics/firstFloor.png"
+	firstCollisionPath      = "assets/csv/firstFloorWalls.csv"
+	firstActivationZonesCSV = "assets/csv/firstFloorZones.csv"
+)
+
+var (
+	// First is the ground level
```
Comment (bcvery1) [inline levels/first.go]: First is not the ground level

File: levels/ground.go [hunk 3]
```diff
@@ -19,6 +19,7 @@ const (
 var (
 	// Ground is the ground level
 	Ground = &Level{
+
```
Comment (bcvery1) [inline levels/ground.go]: Blank line can be removed

File: levels/first.go (hunk 2 above)
Comment (PhantomRaviolis) [reply levels/first.go]: Lol

**Pull request merged**